docs/admin/system-information.rst

```diff
@@ -88,8 +88,8 @@ registered license.
 Cluster settings
 ----------------
 
-The ``sys.cluster.settings`` expression returns information about the currently
-applied cluster settings.
+The ``sys.cluster.settings`` :ref:`expression <gloss-expression>` returns
+information about the currently applied cluster settings.
```

Comment:
Unrelated to your changes, but seems like we missed to add a deprecation note here with the license change to APL? Also in the table below we should note that the values are NULL for CrateDB >= 4.5.